taskweaver/ces/environment.py

```diff
@@ -230,31 +218,29 @@ def start_session(
 
             self._cmd_session_init(session)
             session.kernel_status = "ready"
+        elif self.mode == EnvMode.Container:
+            if platform.system() != "Windows":
+                # change the permission of the session directory and its subdirectories
+                os.chmod(session.session_dir, 0o755)
+                os.chmod(ces_session_dir, 0o755)
+                os.chmod(cwd, 0o755)
```

Comment:
just curious why we need to explicitly configure those folders to 755? (also session dir might not needed if it has been removed below from mounting)

Reply:
session is not needed. for others, I didn't test them but have to make them accessable by taskweaver inside the container in Linux env.